sycl/plugins/esimd_emulator/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ else()
   set(LIBIGFXCMRT_EMU ${LLVM_BINARY_DIR}/pi_esimd_emulator_deps/lib/libigfxcmrt_emu${CMAKE_SHARED_LIBRARY_SUFFIX})
 endif()
 
-set (DEFAULT_CM_EMU_PREBUILT_PACKAGE "https://github.com/intel/cm-cpu-emulation/releases/download/v2022-07-06/intel-cmemu-1.0.23.u20.04-release.x86_64.tar.xz")
+set (DEFAULT_CM_EMU_PREBUILT_PACKAGE "https://github.com/intel/cm-cpu-emulation/releases/download/1.0.23/intel-cmemu-1.0.23.u20.04-release.x86_64.tar.xz")
```

Comment:
There is a newer version of cm emu (Feb 2023): [1.0.31](https://github.com/intel/cm-cpu-emulation/releases/tag/1.0.31).
We need to try using it instead.